File: C/beginner/1096.c
// https://judge.beecrowd.com/pt/problems/view/1096
// Sequência IJ 2

// Libraries
#include <stdio.h>

// Main
int main(){
    // Variables
    int i = 1, j = 7;

    // Iteration
    while(i <= 9){
        printf("I=%d J=%d\n", i, j);
        printf("I=%d J=%d\n", i, j - 1);
        printf("I=%d J=%d\n", i, j - 2);
        i += 2;
    }

    // Ending
    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra

File: Haskell/beginner/1096.hs
-- https://judge.beecrowd.com/pt/problems/view/1096
-- Sequência IJ 2

-- Recursive
sequenceIJ :: Int -> Int -> IO ()
sequenceIJ i j
    | i > 9 = return ()
    | otherwise = do
        putStrLn $ "I=" ++ show i ++ " J=" ++ show j
        putStrLn $ "I=" ++ show i ++ " J=" ++ show (j - 1)
        putStrLn $ "I=" ++ show i ++ " J=" ++ show (j - 2)
        sequenceIJ (i + 2) 7


main = sequenceIJ 1 7

-- By Fernando Serra
-- https://github.com/fernandosserra